### PR TITLE
Bug fix #6048 allign items in add bookmark and edit bookmark layout

### LINKED ISSUE #6048: [Bug] Edit bookmarks screen - fields slightly not aligned
## Steps to reproduce
Go to bookmarks
Long tap one bookmark
Press on edit in the toolbar
Inspect fields and their labels

### Expected behavior
The labels and their fields are aligned

### Actual behavior
The labels and their fields are not aligned by around 3dp (field hint + underline are a bit to the right compared to the label above)

### Device information

* Android device: Samsung S10, Android 9
* Fenix version: 2.1.0, from store

image_2019_10_16T15_16_30_001Z

### PR DESCRIPTION
**Pull request by zaina1986**
Bug fix #6048 allign items in add bookmark and edit bookmark layout

---

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/res/layout/fragment_add_bookmark_folder.xml [hunk 1]
```diff
@@ -14,7 +14,8 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginTop="16dp"
-        android:text="@string/bookmark_name_label"
+        android:layout_marginLeft="4dp"
+        android:text="NAME"
```
Comment (shangeethsivan) [inline app/src/main/res/layout/fragment_add_bookmark_folder.xml]: Do we need to hardcode the string here!  I guess we must use the resource @string/bookmark_name_label

File: app/src/main/res/layout/fragment_add_bookmark_folder.xml [hunk 2]
```diff
@@ -35,7 +36,8 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginTop="8dp"
-        android:text="@string/bookmark_folder_label"
+        android:layout_marginLeft="4dp"
+        android:text="FOLDER"
```
Comment (shangeethsivan) [inline app/src/main/res/layout/fragment_add_bookmark_folder.xml]: Same :: Do we need to hardcode the string here!

**Comment by NotWoods** [comment]
In addition to addressing @shangeethsivan's comments, please attach a screenshot showing the change.

**Comment by ekager** [comment]
I'm going to close this for inactivity

**Pull request closed without merging**